src/multiBrowser.ts

```diff
@@ -0,0 +1,46 @@
+import { When } from '@cucumber/cucumber';
+import { remote } from 'webdriverio';
+
+/**
+ * Open new browser
+ * @param {string} browserName - browser name
+ * @example
+ * When I open new browser as 'browser2'
+ */
+When('I open new browser as {string}', async function (browserName: string) {
+    if (!global.browsers) {
+        global.browsers = {};
+        global.browsers.default = global.browser;
+    }
+    global.browsers[browserName] = await remote(config?.driverConfig);
```

Comment:
is `config` a global variable? Will it open the same browser that was opened previously?

Reply:
yes, it's exposed by CLI

Reply:
so new browser will start with same capabilities